great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -1191,7 +1191,7 @@ def get_profiler(
         key: Union[GeCloudIdentifier, ConfigurationIdentifier]
         if ge_cloud_id:
             key = GeCloudIdentifier(
-                resource_type=GeCloudRESTResource.PROFILER, ge_cloud_id=ge_cloud_id
+                resource_type=GXCloudRESTResource.PROFILER, ge_cloud_id=ge_cloud_id
```

Comment:
update the method parameter too? from ge_cloud_id to gx_cloud_id?

Reply:
sounds good to me!